src/partials/login-tmpl.hbs

```diff
@@ -0,0 +1,24 @@
+
+<div class="login-section">
+    <div class="login-section_info">
+        <h2 class="login-section-heading">Login</h2>
+        <p class="login-section-desc">Get Access to your Orders. Wishlist and Recommendations</p>
+    </div>
+    <form action="#" method="psot" autocomplete="off" class="login-section_form">
```

Comment:
Form action should have a value. If you are making an Ajax request then fetch it from form action and make ajax request. Also method should be get/post.

Reply:
cleaned and rewrote